bindings/go/evmc/evmc.go

```diff
@@ -176,11 +176,11 @@ func (instance *Instance) Destroy() {
 }
 
 func (instance *Instance) Name() string {
-	return C.GoString(instance.handle.name)
+	return C.GoString(C.evmc_instance_name(instance.handle))
```

Comment:
Do not change Go bindings. It will have to create a function wrapper for `evmc_instance_name()`.

Reply:
Too bad, well at least it is maintained by us so it should be fine, but my goal is that external users of EVMC would go through the helpers to detach themselves from the instance internals.

Reply:
Btw, it compiles, so you just mean it is slower because Go generates an extra layer of abstraction?

Reply:
For every C function Go have to generate a wrapper that will switch context from Go runtime to C runtime.

Reply:
Do we consider it too big of an overhead for these two rearely used functions?

Reply:
At the moment yes. I'd have to inspect the generated code. You can leave a TODO comment.

Reply:
If you insist, these two methods should be called once per loading, so it shouldn't be a big deal.